contracts/vendor/Timelock.sol

```diff
@@ -3,33 +3,25 @@ pragma solidity 0.8.15;
 
 import "../ITimelock.sol";
 
-contract BridgeTimelock is ITimelock {
-    /** Custom errors **/
-    error BadDelay();
-    error BadETA();
-    error TransactionExpired();
-    error TransactionNotQueued();
-    error TransactionNotReady();
-    error TransactionReverted();
-    error Unauthorized();
-
-    uint public immutable gracePeriod;
-    uint public immutable minimumDelay;
-    uint public immutable maximumDelay;
+contract Timelock is ITimelock {
```

Comment:
Hm, why is this vendor actually?

Reply:
Oh pulling in with vendoza...

Reply:
yeah, my goals were to reduce the number of changes to Timelock and to track all the things that I changed